api/Avalonia.nupkg.xml

```diff
@@ -1093,6 +1093,12 @@
     <Left>baseline/netstandard2.0/Avalonia.Base.dll</Left>
     <Right>target/netstandard2.0/Avalonia.Base.dll</Right>
   </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:Avalonia.Controls.Embedding.Offscreen.OffscreenTopLevelImplBase.SetFrameThemeVariant(Avalonia.Platform.PlatformThemeVariant)</Target>
```

Comment:
Unrelated, but OffscreenTopLevelImplBase should be [PrivateAPI], something we missed in 11.0.

Reply:
Can be ignored for now, making it PrivateAPI introduces way more changes to this file